pkg/controllers/machinesetsync/machineset_sync_controller.go

```diff
@@ -58,20 +78,21 @@ type MachineSetSyncReconciler struct {
 	MAPINamespace string
 }
 
-// SetupWithManager sets the CoreClusterReconciler controller up with the given manager.
+// SetupWithManager sets up the controller with the Manager.
 func (r *MachineSetSyncReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	infraMachineTemplate, err := getInfraMachineTemplateFromProvider(r.Platform)
 	if err != nil {
-		return fmt.Errorf("failed to get InfraMachine from Provider: %w", err)
+		return fmt.Errorf("failed to get InfraMachineTemplate from Provider: %w", err)
```

Comment:
Should this be `infrastructure machine template`?

Reply:
yup, good catch :D